gpioioctl/gpio.go

```diff
@@ -591,32 +595,85 @@ func (d *driverGPIO) After() []string {
 //
 // https://docs.kernel.org/userspace-api/gpio/chardev.html
 func (d *driverGPIO) Init() (bool, error) {
-	if runtime.GOOS == "linux" {
-		items, err := filepath.Glob("/dev/gpiochip*")
-		if err != nil {
-			return true, err
-		}
-		if len(items) == 0 {
-			return false, errors.New("no GPIO chips found")
+	Chips = make([]*GPIOChip, 0)
```

Comment:
`Chips = nil` would be fine.

Reply:
done. I'm still figuring out zero values in my mind. It bugs me that the zero value for a slice is an empty slice, but the zero value for a map is nil.